workflow/tests/test_workflow.py

```diff
@@ -0,0 +1,57 @@
+import os 
+import subprocess
+from py2cytoscape.data.cyrest_client import CyRestClient
+from py2cytoscape import cyrest
+import yaml
+
+class TestWorkflow:
+
+    @classmethod
+    def setup_method(self):
+        print("setup")
+        self.tps_root = os.getcwd()
+        self.config = os.path.join(
+            self.tps_root,'workflow','tests','test_standards', 'test_default_config.yaml'
+        )
+        self.tps_runner = os.path.join(self.tps_root, 'tps_runner.py')
+
+    @classmethod
+    def teardown_method(self):
```

Comment:
Can you please remind me why we need a teardown for this test?

Reply:
This was originally used to change the working directory after each test.  We will remove it.